k8s.io/Makefile

```diff
@@ -14,6 +14,7 @@ deploy:
 	kubectl apply -f configmap-www-get.yaml
 	kubectl apply -f service-dev.yaml
 	kubectl apply -f deployment.yaml
+	kubectl apply -f certificate-dev.yaml
```

Comment:
I don't think anyone uses the -dev variants.  Can we just nuke them?  In fact we might want to nuke this rule - I forgot it existed

Reply:
deleted!